test/e2e/kubelet_stats.go

```diff
@@ -600,6 +600,40 @@ func (r *resourceMonitor) GetLatest() (resourceUsagePerNode, error) {
 	return result, utilerrors.NewAggregate(errs)
 }
 
+func (r *resourceMonitor) GetMasterNodeLatest(usagePerNode resourceUsagePerNode) resourceUsagePerNode {
+	result := make(resourceUsagePerNode)
+	var masterUsage resourceUsagePerContainer
+	var nodesUsage []resourceUsagePerContainer
+	for node, usage := range usagePerNode {
+		if strings.HasSuffix(node, "master") {
+			masterUsage = usage
+		} else {
+			nodesUsage = append(nodesUsage, usage)
+		}
+	}
+	nodeAvgUsage := make(resourceUsagePerContainer)
+	for _, nodeUsage := range nodesUsage {
+		for c, usage := range nodeUsage {
+			if _, found := nodeAvgUsage[c]; !found {
+				nodeAvgUsage[c] = &containerResourceUsage{Name: usage.Name}
+			}
+			nodeAvgUsage[c].CPUUsageInCores += usage.CPUUsageInCores
```

Comment:
It's not a part of this PR, but maybe you or @yujuhong can change the name to 'AverageCPUUsageInCores' or sth - every second week I'm checking if it's cumulative or an average over an interval, and every second week I forget what I learned...